website/modules/components/EnvironmentHeader.js

```diff
@@ -69,6 +69,14 @@ const Branding = () => (
         React Router
       </Inline>
     </Block>
+    <Block marginTop="10px" flex="1" fontSize="90%" textAlign="center">
+      <a href="/">
+        Learn React from the authors of<br />React Router.{" "}
```

Comment:
The auto formatting is what added the `{" "}` because the space is needed, but yet the code formatting wants it on a new line